auto3dseg/algorithm_templates/swinunetr/configs/hyper_parameters.yaml

```diff
@@ -1,6 +1,7 @@
 amp: true
 bundle_root: monai/apps/auto3dseg/algorithms/templates/SwinUNETR
 ckpt_path: $@bundle_root + '/model_fold' + str(@fold)
+pretrained_path: $'pretrained_model' + '/swin_unetr.base_5000ep_f48_lr2e-4_pretrained.pt'
 data_file_base_dir: /home/yuchengt/Downloads/MONAI-dev/monai/apps/auto3dseg/algorithms/templates/datasets/Task09_Spleen
```

Comment:
Hi @tangy5 , here seems to be a path specific to your machine. Please remove it if it is not necessary.

Reply:
Hi @mingxin-zheng this is not a path only for my local machine, this line specifies a pretrained checkpoint path, then the train script will automatically download a model to this path, the model name is fixed, and will download from web.

Reply:
Sorry @tangy5 , I mean the lines below `data_file_base_dir` and `data_list_file_path`.